src/graphics/meshman.cpp

```diff
@@ -49,8 +49,10 @@ MeshPtr MeshManager::getMesh(rid_t rid) {
 	auto iter = _meshRegistry.find(rid);
 	if (iter == _meshRegistry.end()) {
 		std::unique_ptr<Common::ReadStream> meshResource(ResMan.getResource(rid));
-		if (!meshResource)
+		if (!meshResource) {
+			spdlog::debug("Mesh {} is missing. Falling back to MissingMesh mesh.", rid);
```

Comment:
Small thing, I would make this an error instead of debug, since this is rather important to know.

Reply:
Indeed, I will update this.